homeassistant/components/utility_meter/manifest.json

```
@@ -2,9 +2,16 @@
   "domain": "utility_meter",
   "name": "Utility Meter",
   "documentation": "https://www.home-assistant.io/integrations/utility_meter",
-  "requirements": ["croniter==1.0.6"],
-  "codeowners": ["@dgomes"],
+  "requirements": [
+    "croniter==1.0.6"
+  ],
+  "codeowners": [
+    "@dgomes"
+  ],
   "quality_scale": "internal",
   "iot_class": "local_push",
-  "loggers": ["croniter"]
+  "loggers": [
+    "croniter"
+  ],
+  "config_flow": true
```

(requires running hassfest to re-generate config flows)
```suggestion
  "config_flow": true,
  "integration_type": "helper"
```